templates/problem/voting-controls.html

```diff
@@ -0,0 +1,65 @@
+<hr style="padding-top: 0.1em">
+
+<div class="vote-form" id="id_vote_form_box">
+    <div id="id_has_voted_prompt">
+        <span class="vote-form-info">{{ _('Current Vote:') }}</span>
+        <span id="id_current_vote_value" class="vote-form-value"></span>
+        <br>
+        <br>
+        <span class="vote-form-text">{{ _('Change your vote below!') }}</span>
+        <br>
+        <br>
+    </div>
+    <div id="id_has_not_voted_prompt">
+        <span class="vote-form-text">{{ _("Cast your vote on this problem's weighting below!") }}</span>
+        <br>
+        <br>
+    </div>
+    <form id="id_delete_vote_form">
+        {% csrf_token %}
+    </form>
+    <form id="id_vote_form">
+        {% csrf_token %}
+        <span class="vote-form-info">{{ _('Points') }}</span>
+        <input id="id_points" class="vote-form-value" type="number" step="1"
+               min="{{ min_possible_vote }}" max="{{ max_possible_vote }}" name="points">
+        <br>
+        <div id="id_points_error_box">
+            <br>
+            <span id="id_points_error" class="voting-form-error"></span>
+            <br>
+        </div>
+        <br>
+        <span class="vote-form-info">{{ _('Note') }}</span>
+        <br>
+        <textarea id="id_note" class="vote-form-text" name="note" rows="6" cols="80"></textarea>
```

Comment:
Now that we've discussed why it is optional, can we explicitly mark the textfield as such, maybe in the `Note` part above

Reply:
Added `(Optional)` to the placeholder for note.